bio/megahit/wrapper.py

```
@@ -0,0 +1,106 @@
+
+# Rename/ move output files
+
+# bio/megahit/wrapper.py
+
+# … around line 78 …
+Output_key_mapping = {
+    "contigs": "final.contigs.fa",  # MEGAHIT's output file name
+}
+
+# … later, around line 93 …
+if not has_named_output:
```

Ruff (0.8.2)

Undefined name `has_named_output`

(F821)